starter_code/app.js

```diff
@@ -55,4 +57,29 @@ const index = require('./routes/index');
 app.use('/', index);
 
 
+
+app.get("/movies", (req,res,next)=>{
```

Comment:
This works, however there's a special folder that routes should go in. It helps keep your code more structured, and in your project next week there will be a lot of different routes. Adding everything into app.js gets messy and hard to follow

the routes should be in the index.js file inside the routes folder